client/blocks/login/index.jsx

```diff
@@ -74,7 +83,7 @@ export class Login extends Component {
 
 	render() {
 		const isDisabled = {};
-		if ( this.props.isRequestingLogin ) {
+		if ( this.props.isRequestingLogin || this.state.submitting ) {
```

Comment:
We may not need `isRequestingLogin` now that `submitting` is handling both requests